dev/com.ibm.ws.jdbc/src/com/ibm/ws/rsadapter/DSConfig.java

```diff
@@ -194,6 +245,12 @@
      * JNDI name.
      */
     public final String jndiName;
+    
+    /**
+     * List of identified exceptinos to check if certain errorCode or sqlState values that should map to specific actions.
+     * May be null if none are configured
```

Comment:
The statement that this may be null doesn't appear to be true, and if it were would cause a NullPointerException in the code that follows.